src/common/proto_type.cpp

```diff
@@ -1,87 +1,47 @@
+#include <google/protobuf/struct.pb.h>
```

Comment:
just enforcing cpp/hpp distinction and making a very small refactor. review not necessary.